teachopencadd/talktorials/T016_protein_ligand_interactions/talktorial.ipynb

```diff
@@ -4,58 +4,83 @@
    "cell_type": "markdown",
```

Comment:
<p>I feel like we don't need this quote, can you extract the important numbers (/study design) and mention them shortly in the figure caption instead? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>